### PR TITLE
feat: apply s3 bucket CORS on app start

### PR DESCRIPTION
**Pull request by martinmaillard**

### CODE REVIEW

File: packages/core/env.ts [hunk 1]
```diff
@@ -46,6 +46,7 @@ type ENV_VARS =
   | 'PUBLIC_QUERY_ENDPOINT'
   | 'TRIFID_UI'
   | 'VISUALIZE_UI'
+  | 'UI_BASE'
```
Comment (martinmaillard) [inline packages/core/env.ts:49]: I'm not sure this variable exists in production

**Review by ludovicm67 — APPROVED** [review]
LGTM

**Review by tpluscode — APPROVED** [review]
lgtm

**Comment by martinmaillard** [comment]
Just waiting on https://gitlab.ldbar.ch/vshn/gitops-prod/-/merge_requests/485 to be merged

**Pull request merged**